frontend/app/controllers/Event.scala

```diff
@@ -135,7 +135,11 @@ class Event(
 
   def buy(id: String): Action[AnyContent] = eventbriteService.getEvent(id) match {
       case Some(event@(_: GuLiveEvent)) =>
```

Comment:
I'm just testing whether to collapse this down with the MasterClasses event to be something like:

`case Some(event@(_: LiveEvent)) =>`

Once I've found a advance-booking event I'll comment.